wandb/sdk/lib/redirect.py

```diff
@@ -621,7 +658,7 @@ def uninstall(self):
             return
         self._installed = False
         self.src_wrapped_stream.flush()
-        time.sleep(1)
+        time.sleep(0.5)
```

Comment:
Why are we sleeping at all?  If there's a deterministic reason let's explain it here.  If there are cases where the sleep won't work we should find a solution that always works.